### DISCUSSION TITLE
Support for Groq

### DISCUSSION THREAD

**Discussion by princesychar** [cline/cline#565] [Feature Requests]
i would like for dev to add support for groq and its models

**Comment by meb427** [comment]
Use compatible open AI 
base_url="https://api.groq.com/openai/v1",
https://console.groq.com/docs/openai
That allow you to use it but the models hosted by grok might not work well due to context/capabilities.

**Comment by edwin-truthsearch-io** [comment]
There is a PR for this #3898. Please upvote!
